lib/functions-custom.php

```diff
@@ -49,4 +49,34 @@ function menuActiveClasses($item = null, $type = null, $classes = null) {
 
 function render_embed_caption($caption) {
   echo '<h5 class="text-align-center margin-top-tiny margin-bottom-tiny">' . $caption . '</h5>';
+}
+
+
+function render_quote($text = null, $person = null, $luminary = null) {
```

Comment:
these vars are coming from site options. easily could be a partial instead of a function